doc/source/whatsnew/v0.10.0.rst

```diff
@@ -45,8 +45,7 @@ want to broadcast, we are phasing out this special case (Zen of Python:
 *Special cases aren't special enough to break the rules*). Here's what I'm
 talking about:
 
-.. ipython:: python
-   :okwarning:
+.. code-block:: python
```

Comment:
i think we need to actually paste in the ipython prompts of the execution (e.g. execute the block and paste in the docs)